docs/UserGuide.md

```diff
@@ -278,7 +284,15 @@ Example: Add a <code>John Doe Bespectacled</code> and a <code>John Doe Unadorned
 </div>
 
 Example:
-* `add_student n/John Doe p/98765432 e/johnd@example.com a/John street, Block 123, #01-01 l/Sec2 g/95421323 r/Mother`
+1. You would like to add a new student, `John Doe` to TutorBuddy.
+2. Type in `add_student n/John Doe p/98765432 e/johnd@example.com a/John street, Block 123, #01-01 l/Sec2 g/95421323 r/Mother` 
+   which includes all the essential information the application needs from the student and press <kbd>enter</kbd>.
+
+![add_student_example](images/enhao/add_student%20-%20edited.png)
+
+3. Scrolling down to the end of the student list, you will see that `John Doe` has been added to TutorBuddy.
```

Comment:
Maybe we shouldn't use the code snippet format cause its not a command or response, perhaps bolding would work?

Reply:
Ah right! Agreed! Have updated in the latest commit.